x/genutil/client/cli/export.go

```diff
@@ -70,6 +70,7 @@ func ExportCmd(appExporter servertypes.AppExporter) *cobra.Command {
 				return err
 			}
 			defer cleanup()
+			defer traceWriter.Close()
```

Comment:
This isn't needed, the cleanup does that.

Reply:
@julienrbrt somehow it started getting flagged for this PR and was unrelated, as I documented in the commit message and also in the comments. However, let me revert it. Thanks for the review.

Reply:
Done.